apps/codeforafrica/src/components/ConnectWithMember/ConnectWithMember.js

```diff
@@ -0,0 +1,34 @@
+import { RichTypography } from "@commons-ui/core";
+import Stack from "@mui/material/Stack";
+import React from "react";
+
+import SocialMediaBar from "@/codeforafrica/components/SocialMediaBar";
+import SocialMediaConnectButton from "@/codeforafrica/components/SocialMediaConnectButton";
+
+const ConnectWithMember = React.forwardRef(function ConnectWithMember(
```

Comment:
```suggestion
const ConnectBar = React.forwardRef(function ConnectBar(
```

Since we're going to be using the same component in team members as well as donors/partners page, the name need to be applicable to both.